add step by step for billing (#338)
opengisch/QField-docs: documentation/get-started/tutorials/get-started-qfc.en.md

Change your personal settings. Add a profile picture or get an overview about your ownerships and memberships in organizations.

### Billing

Adding billing information to the account is needed to use QFieldCloud Pro plan or Organization plan. 

For a **Pro plan**, follow these steps :

1. Click on the username up-right the page
2. Click on "Upgrade to pro"
3. In the "Subscription" section, click on "Upgrade"
4. In Pro section, click on "Activate"

> 5. Check the box "I agree with Terms of Service." and click on "Proceed"

Reviewer: ```suggestion
5. Carefully read the "Terms of Service" and agree to the terms of service, then "Proceed"
```

Change your personal settings. Add a profile picture or get an overview about your ownerships and memberships in organizations.

### Billing

Adding billing information to the account is needed to use QFieldCloud Pro plan or Organization plan.

For a **Pro plan**, follow these steps :

1. Click on the username up-right the page
2. Click on "Upgrade to pro"
3. In the "Subscription" section, click on "Upgrade"
4. In Pro section, click on "Activate"
5. Carefully read the "Terms of Service" and agree to the terms of service, then "Proceed"
6. In the "Billing Address" section, fill up the form, and then click on "Proceed to payment"

For an **Organization plan**, follow these steps :

1. Click on the username up-right the page
2. Click on "Create organization"
3. Click on "Create"
4. Choose a name for your organization with only 150 characters or fewer, letters, digits, and @/./+/-/_
5. Click on "Create"
6. Choose how much storage is needed

> 7. Check the box "I agree with the Terms of Service." and click on "Proceed"

Reviewer: @lucienicolier can please also update here?

Author: @m-kuhn done
